src/screens/OrganizationActionItems/ItemModal.tsx

```
@@ -226,6 +226,16 @@ const ItemModal: FC<InterfaceItemModalProps> = ({
     field: keyof InterfaceFormStateType,
     value: string | number | boolean | Date | undefined | null,
   ): void => {
+    // Special handling for allottedHours
+    if (field === 'allottedHours') {
+      // If the value is not a valid number or is negative, set to null
+      const numValue = typeof value === 'string' ? Number(value) : value;
+      if (typeof numValue !== 'number' || isNaN(numValue) || numValue < 0) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)